code/modules/jobs/job_types/assistant.dm

```diff
@@ -16,11 +16,7 @@ Assistant
 
 
 /datum/job/assistant/get_access()
-	if((config.jobs_have_maint_access & ASSISTANTS_HAVE_MAINT_ACCESS) || !config.jobs_have_minimal_access) //Config has assistant maint access set
-		. = ..()
-		. |= list(access_maint_tunnels)
-	else
-		return ..()
+	return ..()
```

Comment:
While you are somewhat right, I don't think it should ever be enabled, but I might be biased. As I said in another comment, maintenance should be reserved for personnel who HAVE to do something there.